src/main/java/com/hokka1do/bookme/professionals/Professional.java

```diff
@@ -0,0 +1,33 @@
+package com.hokka1do.bookme.professionals;
+
+import org.springframework.web.bind.annotation.GetMapping;
+
+import javax.persistence.*;
```

Comment:
Wildcard imports should be avoided
https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad

Reply:
Got it. Will check in a future.